Rename Method as part of Refactoring

bharatwaaj opened the pull request:
<h1>Refactoring: Rename Method</h1> 

<h3> Reason for change: </h3>
The method selects a specified element based on the parameterized condition. So, the function name must be clearly mentioned as selectSpecifiedElement rather than just selects.

&nbsp;
**From** select
**To** selectSpecifiedElement

&nbsp;

Particulars | Details | Comments
-- | -- | --
Commit Hash | com.gargoylesoftware.htmlunit |  
Branch Name | asdc-assignment-set-1-extract-method |  
Branch Link | https://github.com/bharatwaaj/htmlunit/tree/asdc-assignment-set-1-rename-method |  
PR URL | https://github.com/HtmlUnit/htmlunit/pull/465 |  

bharatwaaj committed:
Update CssStyleSheet.java

rbri commented on src/main/java/com/gargoylesoftware/htmlunit/css/CssStyleSheet.java:
```diff
@@ -353,8 +353,8 @@ else if (bomIn.hasBOM(ByteOrderMark.UTF_16LE)) {
      */
-    public static boolean selects(final BrowserVersion browserVersion, final Selector selector,
-            final DomElement element, final String pseudoElement, final boolean fromQuerySelectorAll) {
+    public static boolean selectsSpecifiedElement(final BrowserVersion browserVersion, final Selector selector,
```
you are right, better naming might be a good idea but this method is public and i do not want to break the interface here.

rbri commented on src/main/java/com/gargoylesoftware/htmlunit/css/CssStyleSheet.java:
```diff
@@ -353,8 +353,8 @@ else if (bomIn.hasBOM(ByteOrderMark.UTF_16LE)) {
-    public static boolean selects(final BrowserVersion browserVersion, final Selector selector,
-            final DomElement element, final String pseudoElement, final boolean fromQuerySelectorAll) {
+    public static boolean selectsSpecifiedElement(final BrowserVersion browserVersion, final Selector selector,
+                                                  final DomElement element, final String pseudoElement, final boolean fromQuerySelectorAll) {
```
I guess this reformating breaks the formating rules

rbri commented:
see comments, please revert the renaming, and please add your name to the contributor list in the pom,

bharatwaaj committed:
Update CssStyleSheet.java

bharatwaaj committed:
Update pom.xml

bharatwaaj commented:
> see comments, please revert the renaming, and please add your name to the contributor list in the pom,

Hi, I have reverted the renaming and added my name to contributor list in the pom.xml, thanks a ton!

bharatwaaj committed:
Update CssStyleSheet.java

rbri commented:
Many thanks
